Application/src/Utils/TestImageComponent.js

```diff
@@ -56,6 +56,32 @@ function TestImageComponent() {
         <img alt='Test bottomMiddle' src={imageGrid.images.bottomMiddle.img} style={imageStyles} />
         <img alt='Test bottomRight' src={imageGrid.images.bottomRight.img} style={imageStyles} />
       </div>
+      <div
+        style={{
+          height: '50%',
+          width: '50%',
+          justifyContent: 'center',
+          alignItems: 'center',
+          flexDirection: 'column',
+          display: 'flex',
+        }}
+      >
+        <img
+          alt='Test top'
+          src={imageGrid.images.tempImages.top.img}
+          style={{ width: '100%', height: '90%', margin: '1px' }}
+        />
+        <img
+          alt='Test middle'
+          src={imageGrid.images.tempImages.middle.img}
+          style={{ width: '100%', height: '90%', margin: '1px' }}
+        />
+        <img
+          alt='Test bottom'
+          src={imageGrid.images.tempImages.bottom.img}
+          style={{ width: '100%', height: '90%', margin: '1px' }}
+        />
+      </div>
```

Comment:
I absolutely love the work you have done here! let's see if we can potentially include this react component into the finally project.